form/form_collections.rst

```diff
@@ -746,6 +746,86 @@ the relationship between the removed ``Tag`` and ``Task`` object.
     you'll need to do extra work to make sure that the relationship is properly
     updated (whether you're adding new tags or removing existing tags) on
     each Tag object itself.
+    
+    You can optionnally reduce the code in your controller and make it reusable to handle other embedded form.
+    First create a helper
+    
+        // App\Helper;
+        use Doctrine\Common\Collections\ArrayCollection;
+        use Doctrine\ORM\EntityManagerInterface;
+        
+        class Helper      
+          private $em;
```

Comment:
The code indentation is not correct. The build is failing because of this. Can you correct this by using 4 spaces:

```
class Helper      
    private $em;
```

This also includes other indentation errors in your changes.